src/api/content-rest-api/api/tags.api.ts

```diff
@@ -384,5 +384,39 @@ parameter are returned in addition to those specified in the **fields** paramete
             pathParams, queryParams, headerParams, formParams, postBody,
             contentTypes, accepts , TagEntry);
     }
+/**
+    * Delete a tag
+    *
+    * Deletes a tag by **tagId**. This will cause the tag to be removed from all nodes.
+    *
+    * @param tagId The identifier of a tag.
+    * @return Promise<{}>
+ */
+    deleteTag(tagId: string): Promise<any> {
```

Comment:
I think there is no real response after removing so type can be void instead of any.

Reply:
changed to void